### ISSUE TITLE
Multiple record types in a request

### ISSUE THREAD

**Issue by fschiettecatte** [ga4gh-discovery/ga4gh-case-discovery#16]
Do we want to support multiple record types in a request?

**Comment by Relequestual** [comment]
Blocked by #24

**Comment by Relequestual** [comment]
Given the previous blocker is resolved in favour of using the component system, this issue is no longer relevant.
